features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -67,10 +67,10 @@ const DigitalGiantsStory = () => {
                 title={i18n.t(`${i18nHeaderCommon}:section.dataTypes`)}
             />
             <Tabs>
-                <Tab label="Hello">
+                <Tab label="Hello" translation="Hallo">
```

Comment:
Did you consider using i18n here? This can become unwieldy eventually, with many different files to change.

Reply:
When I replace the dummy data, I will absolutely use i18n here! Just wanted to have an example for the others working on this, how you add translations.

Reply:
But... why? polyLook doesn't have any translation logic currently - it doesn't know the users language, and it doesn't have any language based logic. Even in this case, if _translation_ is there, it simply uses it, regardless of the user's locale. Why wouldn't we just pass in a translated string as label, always?

Reply:
You mean, why do we have a `label` and `translation` instead of simply using silly-i18n on the label?

Reply:
Sorry didn't see the comment just now as it was resolved. The label in a component is also the id and in composed components with a chart for example in the tab content it is most comfortable to use than having translations as id

Reply:
So, if I understand this correctly, `label` is _not_ the label, it's actually the `id`. Likewise, `translation` is _not_ the translation, it's actually the label.

I guess you see where I'm going :P

Reply:
That's pretty much correct yeah :D